### DISCUSSION TITLE
Trigger other bots

### DISCUSSION THREAD

**Discussion by jbaumgarten** [Rapptz/discord.py#9242] [Q&A] [answered]
Hi guys,

I have read it isn't possible to trigger slashcommands with our bot but the posts are really old.
Is it still the case?
There is no way to workaround?

Yours sincerely
Julien

**Comment by cibere** [comment, marked as answer]
You are correct, bots are unable to invoke slash commands. This has not changed.

**Reply by jbaumgarten** [reply]
Ok thx. I hopped it has changed :D
